### PR TITLE
Eg 9 vs animation

### LINKED ISSUE #9: Create a "VS" animation
**Is your feature request related to a problem? Please describe.**
When two Pokemon are searched, show an animated "VS" pop up on screen between the two Pokemon.

### PR DESCRIPTION
**Pull request by BlackCubes**
## Changes
1. Created a Versus component in Versus.js.
2. Added a setTimeout with useEffect to decide whether to display certain parts of the Versus component.
3. Added isVersus states in App.js.
4. Added setIsVersus state in getPokeByNameOrIdAPI function in App.js.
5. Passed isVersus state from App.js to Main component as prop.
6. Passes isVersus prop from Main.js to Versus component as prop.
7. Passed setIsVersus state from App.js to Search component as prop.
8. Added setIsVersus in conditional of removePokemon function which resets the Versus component.
2. Created a Versus.css style file.
3. Styled the Versus component in Versus.css.

## Purpose
The purpose of this feature was to add a versus animation when the user selects an enemy pokemon.

## Approach
This approach addresses the problem of creating a better UI experience for the user.

Closes #9

### CODE REVIEW

File: pokemon-app/src/styles/Versus.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  50% {
+    transform: translate(84.6%, -33.5%);
+    opacity: 1;
+  }
+
+  to {
+    transform: translate(84.6%, -33.5%);
+    opacity: 0;
+  }
+}
```
Comment (Lap343) [inline pokemon-app/src/styles/Versus.css:222]: VS animation looks soooooo good

**Review by Lap343 — APPROVED** [review]
Code looks great

File: pokemon-app/src/styles/Versus.css (hunk 1 above)
Comment (BlackCubes) [reply pokemon-app/src/styles/Versus.css:222]: Thank you Team Captain 🙇‍♂️!

**Review by connielion — APPROVED** [review]
Nice! Can't wait to see the app.

**Review by Wolftownn — APPROVED** [review]
Soooo good!

**Pull request merged**